<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="208374502">
    <data-source source="LOCAL" name="Sqlite (Xerial) - identifier.sqlite" uuid="6dc90594-5486-48c5-9e86-c158002889e0">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:identifier.sqlite</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="6275acaa-3c75-49da-918c-b4edc66b3881">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/food_site/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
    </data-source>
  </component>
</project>